packages/contracts/contracts/L1/ValidatorManager.sol

```diff
@@ -620,6 +620,10 @@ contract ValidatorManager is ISemver, IValidatorManager {
      * @return The amount of reward from commission and base reward for the validator.
      */
     function _calculateReward(address validator) internal view returns (uint128, uint128, uint128) {
+        if (validator == ASSET_MANAGER.SECURITY_COUNCIL()) {
+            return (0, 0, BASE_REWARD);
```

Comment:
Is there a reason to return `BASE_REWARD` to `validatorReward` and not to `baseReward`?

Reply:
There are no differences between two ways, baseReward is meant to be distributed to delegators, so I returned it as `validatorReward`.